editor/src/components/canvas/controls/follow-mode/follow-mode-hooks.tsx

```diff
@@ -0,0 +1,11 @@
+import type { MouseCallbacks } from '../select-mode/select-mode-hooks'
+import { NO_OP } from '../../../../core/shared/utils'
+import { useKeepShallowReferenceEquality } from '../../../../utils/react-performance'
+
+export function useFollowModeSelectAndHover(): MouseCallbacks {
+  return useKeepShallowReferenceEquality({
```

Comment:
the minorest of nitpicks, but useKeepShallowReferenceEquality is probably slower than React.useMemo, which is slower than storing the return value in a global const in the module root

Reply:
Makes sense. In https://github.com/concrete-utopia/utopia/pull/4506/commits/2c1813f2c35af51994950ebd433900cc41e4aed5 I kept it as a hook for consistency, but returning a const. Would that work for you?